src/Options/index.js

```diff
@@ -553,6 +553,15 @@ export interface DatabaseOptions {
   enableSchemaHooks: ?boolean;
   /* The duration in seconds after which the schema cache expires and will be refetched from the database. Use this option if using multiple Parse Servers instances connected to the same database. A low duration will cause the schema cache to be updated too often, causing unnecessary database reads. A high duration will cause the schema to be updated too rarely, increasing the time required until schema changes propagate to all server instances. This feature can be used as an alternative or in conjunction with the option `enableSchemaHooks`. Default is infinite which means the schema cache never expires. */
   schemaCacheTtl: ?number;
+  /* Disables behavior to ensure case-insensitive index on field username on _User collection. Set to `true` if using a database not supporting case-insensitive indexes.
+  :DEFAULT: false */
+  disableEnsureUsernameCaseInsensitiveIndex: ?boolean;
+  /* Disables behavior to ensure case-insensitive index on field email on _User collection. Set to `true` if using a database not supporting case-insensitive indexes.
+  :DEFAULT: false */
+  disableEnsureEmailCaseInsensitiveIndex: ?boolean;
+  /* Disables behavior to ensure time to live index on field expire on _Idempotency collection. Set to `true` if using a database not supporting TTL index on this field.
+  :DEFAULT: false */
+  disableEnsureIdempotencyExpireIndex: ?boolean;
```

Comment:
It will introduce security issues to just disable case insensitivity on username and email, i sent a more complete PR to force email and username to lowercase in this cases: https://github.com/parse-community/parse-server/pull/8494

Reply:
Thanks, I noticed this comment just now
I would update my PR to take your changes once your PR is merged

Reply:
> It will introduce security issues to just disable case insensitivity on username and email, i sent a more complete PR to force email and username to lowercase in this cases: #8042

Thanks, I noticed this comment just now
I would update my PR to take your changes once your PR is merged